content/dstu2/diagnostic-report.md

```diff
@@ -0,0 +1,26 @@
+---
+title: DiagnosticReport | FHIR DSTU 2 BALLOT API
+---
+
+# Immunization
```

Comment:
@davischristi We all missed this when reviewing the PR, but the title here should be Diagnostic Report